public/sitemap.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<sitemapindex xmlns="http://www.sitemaps.org/schemas/sitemap/0.9">
+<sitemap><loc>https://esp.ethereum.foundation/sitemap-0.xml</loc></sitemap>
```

Comment:
Why does this sitemap link to sitemap-0.xml? why not just have one file sitemap.xml with all our urls?

Reply:
This is the [index sitemap](https://github.com/iamvishnusankar/next-sitemap#index-sitemaps) generated automatically by the tool